bundles/org.openhab.core.semantics/src/main/java/org/openhab/core/semantics/model/equipment/AlarmSystem.java

```diff
@@ -17,7 +17,7 @@
 import org.openhab.core.semantics.TagInfo;
 
 /**
- * This class defines a Alarm System.
```

Comment:
Maybe it is possible to apply the rules in the generator as well? 

https://github.com/openhab/openhab-core/blob/af3738487c5e214165191fe316cfbee02c9f1578/bundles/org.openhab.core.semantics/model/generateTagClasses.groovy#L82

Otherwise it complicates generation and the changes may be overwritten the next time the classes are generated.

You can test generation by activating the profile by adding `-DgenerateTagClasses` to a build.

Reply:
I am not going to do this, brcause I jave not setup a build environment.

Reply:
It's very easy to do without a  build environement, see: https://github.com/dilyanpalauzov/openhab-core/pull/1 :slightly_smiling_face:

Reply:
The change proposed by you is only relatively good, as before U and H both A and AN can be used.

I suggest my PR and your PR are handled separatedly.